### PR TITLE
[tune] Use latest Ax release

### PR DESCRIPTION
**Pull request by krfricke**
## Why are these changes needed?

Ax released a new version but didn't push it to PyPI, yet. See https://github.com/facebook/Ax/issues/394

In the meantime we'll pin the version to the latest release on Github.

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: python/requirements_tune.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-ax-platform
+# ax-platform
+git+https://github.com/facebook/Ax.git@0.1.16
```
Comment (sumanthratna) [inline python/requirements_tune.txt:2]: not sure but do we want to use `git+https://github.com/facebook/Ax.git@0.1.16#egg=ax-platform`?

Comment (krfricke) [reply python/requirements_tune.txt]: Should work without it, but I updated the PR to make it explicit. Thanks!

**Review by sven1977 — APPROVED** [review]
Thanks for the fix.

**Comment by richardliaw** [comment]
Looks like Horovod is broken right now too

**Pull request merged**